en/docs/visual-programming/sequence-diagram-view.md

```diff
@@ -1,24 +1,28 @@
-# Sequence diagram view
-With Ballerina, you can view your code in a graphical mode and develop your program logic through it. This
-allows you to look at your program logic holistically and gives you the ability to comprehend the program at 
-a glance. This visual representation has its roots in sequence diagrams and can capture how the logic of
-the program flows, how the concurrent execution flow works, which remote endpoints are involved, and how they interact 
-with the different objects in the program.
+# Diagram view
```

Comment:
Not very sure about this

Reply:
IMO, `Sequence diagram view` is correct as this page is only about the sequence diagram aspects of the visual diagrams provided by VS code extension. Data mapper and project design view are the other main two diagrams where we have separate pages for them.

Reply:
Then we will make it `Sequence diagram view` instead of `Sequence Diagram view` (capitalization). Hope that is okay.

Reply:
Kept it as `Sequence Diagram` for now for consistency